ssl/record/ssl3_record.c

```diff
@@ -678,9 +678,7 @@ int tls1_enc(SSL *s, int send)
                      * we can't write into the input stream: Can this ever
                      * happen?? (steve)
                      */
-                    fprintf(stderr,
-                            "%s:%d: rec->data != rec->input\n",
-                            __FILE__, __LINE__);
+                    fprintf(stderr, "tls1_enc(): rec->data != rec->input\n");
```

Comment:
FWIW, the description did say that is was *mostly* a simple search+replace, with a few instances changed to just print the function name instead. This and the one in dtls_read_failed() that you also noted were what was meant by the latter part of that sentence.
The commit comment did further note that these instances should probably die anyway...